tests/python/quantization/test_quantization.py

```diff
@@ -713,6 +713,32 @@ def forward(self, x):
         check_quantized_fc((256, 2048, 2, 2), 800, False, qdtype)
         check_quantized_fc((256, 111, 2, 2), 800, False, qdtype)
 
+@use_np
+def test_quantized_transpose():
```

Comment:
Maybye add some test for quantized transpose after some other quantizable operator e.g. conv, fc.